docs/components/demo/TimeRangeField/css/styles.css

```
@@ -0,0 +1,56 @@
+
+.TimeField {
+  display: flex;
+  padding: 0.5rem;
+  align-items: center;
+  border-radius: 0.25rem;
+  border-width: 1px;
+  text-align: center;
+  background-color: #ffffff;
+  user-select: none;
+  color: var(--green-10);
+  border: 1px solid var(--gray-9);
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-width



(lint/suspicious/noShorthandPropertyOverrides)